### ISSUE TITLE
[Bug] Custom Format spacing for the Clock applet is inconsistent between reboots on a vertical panel

### ISSUE THREAD

**Issue by justinhschaaf** [BuddiesOfBudgie/budgie-desktop#357]
### Description

I have my desktop setup so the default panel is vertical on the right side of the screen. Upon setting a Custom Format for the Clock applet on this panel and restarting the system, extra spacing is added to the bottom of the applet that isn't present in the Custom Format text box in the applet's settings.

### Budgie version

10.7.1

### Operating System

Fedora 38

### Steps to reproduce the issue

1. In the Budgie Desktop Settings app, change the position on the default bottom panel to the right edge of the screen.
2. In the Clock applet for the panel, enable the Custom Format toggle and change the format. I used `%H:%M %S%n%n %Y %m.%d`.
3. Reboot the system.

### Actual result

Before Reboot:

![Budgie Clock Applet Before Reboot](https://user-images.githubusercontent.com/26681724/233763392-92eb2a3e-722f-4194-9de0-e994dfe4d4fd.png)

After Reboot:

![Budgie Clock Applet After Reboot](https://user-images.githubusercontent.com/26681724/233763401-6953740c-bcd4-433d-a234-adfd9dad41cd.png)

### Expected result

The spacing for the custom format on the clock applet should be consistent between reboots.

**Comment by serebit** [comment]
I tried to reproduce this on Arch (same GTK theme, same format, etc) to no avail. Is this still happening on your machine?

**Comment by justinhschaaf** [comment]
Yep, still running into it. As a test, I just tried deleting the custom format value in dconf Editor and setting it again only to run into the same problem. My guess as to what might've broken it was that I experimented with adding newlines (`%n`) to the end of the format when I first made it in an attempt to add some padding below the clock; however, since these didn't apply in real time when changing the format, I just removed them and didn't think much of it. I thought that maybe these newline characters might still be there somehow, and as such tried deleting the key and set the format again to no avail; I'm not an expert with how Linux works and have been daily driving it for less than a year at this point, so I'm not sure how I'd look into the issue further on my own either.
